Configuration/FlexForms/Pi1.xml

```diff
@@ -112,7 +112,19 @@
 							</config>
 							<displayCond>FIELD:switchableControllerActions:=:Topic->list</displayCond>
 						</TCEforms>
-					</settings.listTopics>                    
+					</settings.listTopics>
+					<settings.maxTopicItems>
+						<TCEforms>
+							<label>LLL:EXT:typo3_forum/Resources/Private/Language/locallang_flexforms.xml:Display_MaxTopicItems</label>
+							<config>
+								<type>input</type>
+								<size>6</size>
+								<eval>int,required</eval>
+								<default>3</default>
```

Comment:
Since the previous default value was hardcoded to 6, I'd prefer to keep that value as default, here. Otherwise this will break existing installations on upgrade.